rust/processor/src/schema.rs

```diff
@@ -518,6 +539,80 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    fungible_asset_activities (transaction_version, event_index) {
+        transaction_version -> Int8,
+        event_index -> Int8,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 66]
+        storage_id -> Varchar,
+        #[max_length = 1000]
+        asset_type -> Varchar,
+        is_frozen -> Nullable<Bool>,
```

Comment:
Does the activity cover freezing/unfreezing a fungible store or coin store as well?

Reply:
CoinStore has freezing. You can freeze CoinStores (but not other structs that contain Coin)

Reply:
Hmm. I guess there is no event. Good point for balances though.